### PR TITLE
Update using-webhooks-with-github-apps.md to replace offline "Smee"

### PR DESCRIPTION
**Pull request by alexbouchardd**
_DISCLAIMER: I'm Hookdeck co-founder and this specific documentation page was brought to my attention by a user using Hookdeck for their Github webhooks on 3 different apps._

### Why:

Smee.io seems no longer operational and the domain can no longer be reached leading to broken links. Hookdeck Console & CLI is an alternative to Smee that doesn't require authentication or payments. 

This specific piece of documentation was shared with me by a user who proposed that I open this PR so that other developers would be guided toward a working solution.

Closes:  N/A, should I open an issue for the "broken" link?

### What's being changed (if available, include any code snippets, screenshots, or gifs):

The "Smee" tool referral was changed to "Hookdeck" tool as a maintained replacement for Smee functionality (preview webhooks, localhost forwarding).

Hookdeck has a "development" tool and a "production" tool. The former is being proposed as an alternative to Smee. You can evaluate it at the link provided in the documentation: https://console.hookdeck.com. It does not require an account or payment. The latter is both a free and paid product for production use cases. I included a reference to it as well since other products such as Azure Functions and AWS Lambda are mentioned. I'll let you judge if that's appropriate for these docs but Hookdeck is used in production by multiple Github apps.

For reference here's the Hookdeck CLI Repo https://github.com/hookdeck/hookdeck-cli

### Check off the following:

- [x] I have reviewed my changes in staging (look for the "Automatically generated comment" and click the links in the "Preview" column to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

**Comment by alexbouchardd** [comment]
Bumping for the stale label. Hoping a maintainer will have a look! :)

**Comment by alexbouchardd** [comment]
Thanks for the update @cmwilson21. No worries at all. Let me know if there's any way I can help.

**Comment by skedwards88** [comment]
Thanks for opening this issue @alexbouchardd ! Can you clarify more what you mean by "Smee.io seems no longer operational and the domain can no longer be reached leading to broken links"? https://smee.io/ loads for me currently.

**Comment by alexbouchardd** [comment]
@skedwards88 Oh! The DNS wasn't resolving for multiple hours/days while I worked on this PR, and the user reported the issue. The GitHub repo also didn't seem very active, so I assumed it was taken offline for good.

I'll let you decide if the Console is a better solution, is any case happy to edit or close that PR based on your opinion on that front. Do you think the Console would provide value to Github users & documentation readers?

**Comment by skedwards88** [comment]
No worries! I figured it was something like that. My main concern about changing from smee to hookdeck in this article is that we use smee throughout the docs. For user simplicity, it is probably good to stay consistent. What do you think of keeping this article focused on smee but instead listing hookdeck as an alternative, like we do for ngrok?

**Comment by alexbouchardd** [comment]
@skedwards88 Sounds totally reasonable to me! Should I propose a new edit or will you tackle it?

**Comment by skedwards88** [comment]
A new edit from you would be perfect 🙇 Feel free to modify this PR or close it and open a fresh PR. Thanks for your flexibility!

File: content/apps/creating-github-apps/creating-github-apps/using-webhooks-with-github-apps.md
Comment (skedwards88) [inline content/apps/creating-github-apps/creating-github-apps/using-webhooks-with-github-apps.md:30]:
```suggestion
While you develop and test your app, you can use a webhook payload delivery service like [Smee](https://smee.io/) to capture and forward webhook payloads to your local development environment. Never use Smee for an application in production, because Smee channels are not authenticated or secure. Alternatively, you can use a tool like [ngrok](https://dashboard.ngrok.com/get-started), [localtunnel](https://localtunnel.github.io/www/), or the [Hookdeck Console](https://console.hookdeck.com?provider=github) that exposes your local machine to the internet to receive the payloads.
```

File: content/apps/creating-github-apps/creating-github-apps/using-webhooks-with-github-apps.md
Comment (skedwards88) [inline content/apps/creating-github-apps/creating-github-apps/using-webhooks-with-github-apps.md:44]:
```suggestion
To handle a higher volume of webhook traffic for a large app in production, consider using asynchronous webhook handling on a dedicated server. You can achieve this by employing a queue, where the webhook handler pushes data to the queue, and separate processes perform subsequent actions based on the events. Additionally, you can use cloud functions such as [Azure Functions](https://azure.microsoft.com/en-us/products/functions/), [AWS Lambda](https://aws.amazon.com/lambda/), or [Hookdeck](https://hookdeck.com) to help scale the app for handling large volumes of webhook events.
```

**Review by skedwards88 — COMMENTED** [review]
Thanks for these changes! I'm just going to add a couple of commas to align with the style guide, then we'll get this merged for you.

**Review by skedwards88 — APPROVED** [review]

**Comment by alexbouchardd** [comment]
Awesome, thanks @skedwards88!

**Comment by mehran636311** [comment]
> _DISCLAIMER: I'm Hookdeck co-founder and this specific documentation page was brought to my attention by a user using Hookdeck for their Github webhooks on 3 different apps._
> 
> ### Why:
> Smee.io seems no longer operational and the domain can no longer be reached leading to broken links. Hookdeck Console & CLI is an alternative to Smee that doesn't require authentication or payments.
> 
> This specific piece of documentation was shared with me by a user who proposed that I open this PR so that other developers would be guided toward a working solution.
> 
> Closes: N/A, should I open an issue for the "broken" link?
> 
> ### What's being changed (if available, include any code snippets, screenshots, or gifs):
> The "Smee" tool referral was changed to "Hookdeck" tool as a maintained replacement for Smee functionality (preview webhooks, localhost forwarding).
> 
> Hookdeck has a "development" tool and a "production" tool. The former is being proposed as an alternative to Smee. You can evaluate it at the link provided in the documentation: https://console.hookdeck.com. It does not require an account or payment. The latter is both a free and paid product for production use cases. I included a reference to it as well since other products such as Azure Functions and AWS Lambda are mentioned. I'll let you judge if that's appropriate for these docs but Hookdeck is used in production by multiple Github apps.
> 
> For reference here's the Hookdeck CLI Repo https://github.com/hookdeck/hookdeck-cli
> 
> ### Check off the following:
> * [x]  I have reviewed my changes in staging (look for the "Automatically generated comment" and click the links in the "Preview" column to view your latest changes).
> * [x]  For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

Ok

**Pull request merged**
